src/controls/TeamPicker/TeamPickerStyles.ts

```diff
@@ -7,9 +7,11 @@ import {
   mergeStyleSets,
 } from "office-ui-fabric-react/lib/Styling";
 import { Theme } from "spfx-uifabric-themes";
```

Comment:
seems like unused `import`.
Could you check for other unused `imports`, if any?

Reply:
changed to selective import